resources/task.rb

```diff
@@ -43,6 +43,7 @@
 attribute :start_day, kind_of: String, default: nil
 attribute :start_time, kind_of: String, default: nil
 attribute :day, kind_of: [String, Integer], default: nil
+attribute :months, kind_of: String, default: nil
```

Comment:
I was mainly trying to stay consistent with the way the day frequency was written.